### ISSUE TITLE
browserify require is broken when jQuery versions mismatch

### ISSUE THREAD

**Issue by jeremylevy** [cloudfour/hideShowPassword#39]
Hello,

Just encountered a problem when requiring `hideshowpassword` using Browserify.

Given that I need to support IE8, jQuery is defined as `^1.11.3` in my `package.json`.

Since you have also added jQuery as dependency, and that the two versions mismatch, the `hideshowpassword` function will be attached to your jQuery instance, instead of mine, making it unavailable from my code.

Why jQuery has been added as production dependency, given that the function is attached to `$.fn` ?

**Comment by tylersticka** [comment]
I can confirm when using an earlier version of jQuery than what's defined in this module's `package.json` that two jQuery libraries are bundled. I'll investigate further, I suspect our `package.json` will need to be updated.

**Comment by tylersticka** [comment]
@jeremylevy I have a potential fix pushed to the `fix-jquery-dep-ver` branch. You can try this locally by removing your current `hideshowpassword` dependency, and re-installing with the following command:

```
$ npm install --save cloudfour/hideShowPassword#fix-jquery-dep-ver
```

You may also need to run `npm dedupe`. After that, you _should_ no longer see duplicate jQuery modules in the bundle that Browserify generates.

Can you confirm whether or not this resolves the issue? If so, I'll go ahead and merge the branch and push this update to `npm`.

Thanks for your patience!

**Comment by jeremylevy** [comment]
It works! Your rock! Thanks :+1:

**Comment by tylersticka** [comment]
Great! I'll go ahead and merge and bump the version...

**Comment by tylersticka** [comment]
@jeremylevy This fix is available in version `2.0.8`. I just pushed an update to `npm`.

Thanks again!
